src/main/webapp/script.js

```diff
@@ -13,16 +13,165 @@
+        '\"As you know, life is an echo; we get what we give.\" - David DeNotaris', 
+        '\"There are no regrets in life, just lessons.\" - Jennifer Aniston', 
+        '\"I believe that nothing in life is unimportant every moment can be a beginning.\" - John McLeod', 
+        '\"Find people who will make you better.\" - Michelle Obama', 
+        '\"As my knowledge of things grew I felt more and more the delight of the world I was in.\" - Helen Keller', 
+        '\"Benjamin Franklin was a humanitarian that dedicated his life to making contributions to all humans. He had a clear purpose for himself: improve the human race.\" - Paulo Braga', 
+        '\"You cannot control everything that happens to you; you can only control the way you respond to what happens. In your response is your power.\" - Anonymous', 
+        '\"Don’t allow your past or present condition to control you. It’s just a process that you’re going through to get you to the next level.\" – T.D. Jakes', 
+        '\"You will meet two kinds of people in life: ones who build you up and ones who tear you down. But in the end, you’ll thank them both.\" - Anonymous', 
+        '\"My mission in life is not merely to survive, but to thrive; and to do so with some passion, some compassion, some humor, and some style.\" – Maya Angelou', 
+        '\"If we don’t change, we don’t grow. If we don’t grow, we aren’t really living.\" – Gail Sheehy', 
+        '\"You choose the life you live. If you don’t like it, it’s on you to change it because no one else is going to do it for you.\" – Kim Kiyosaki', 
+        '\"It is impossible to live without failing at something, unless you live so cautiously, that you might as well not have lived at all – in which case you fail by default.\" - Anonymous', 
+        '\"Life doesn’t require that we be the best, only that we try our best.\" – H. Jackson Brown Jr.', 
+        '\"The way I see it, every life is a pile of good things and bad things. The good things don’t always soften the bad things, but vice versa, the bad things don’t always spoil the good things and make them unimportant.\" – Doctor Who', 
+        '\"Life isn’t about waiting for the storm to pass, it’s about learning to dance in the rain.\" – Vivian Greene', 
+        '\"I enjoy life when things are happening. I don’t care if it’s good things or bad things. That means you’re alive.\" – Joan Rivers', 
+        '\"There’s more to life than basketball. The most important thing is your family and taking care of each other and loving each other no matter what.\" – Stephen Curry', 
+        '\"Today, you have 100% of your life left.\" – Tom Landry (Football Quotes)', 
+        '\"Nobody who ever gave his best regretted it.\" – George Halas', 
+        '\"Make each day your masterpiece.\" – John Wooden', 
+        '\"You can’t put a limit on anything. The more you dream, the farther you get.\" – Michael Phelps', 
+      ];
```

Comment:
Consider putting all of the strings in a separate file and read it in line by line. It could clean up this file so its easier to read/work on. It would also reduce the memory needed to store  a large array of strings like this every time the page is loaded. It's probably fine for now if it just stays at the bottom of the page, but if there was a larger array, it could become a little unwieldy.

Reply:
I have moved the quotes into a servlet file to prevent them from being loaded each time the script is run.